package.json

```diff
@@ -212,7 +212,9 @@
     "resolutions": {
         "@types/node": "^16.11.7",
         "ansi-regex": "^5.0.1",
-        "dot-prop": "^5.2.0",
+        "**/compare-func/dot-prop": "^5.2.0",
```

Comment:
Changed this line because we have some dependencies that need dot-prop v6. Resolving to ^5.2.0 forces those dependencies to downgrade, but resolving to >=5.2.0 resolves all versions to v7, which is incompatible with the packages that depend on v5.